distgo/config/configproduct.go

```diff
@@ -102,6 +102,7 @@ func (cfg *ProductConfig) ToParam(productID distgo.ProductID, scriptIncludes str
 	}
 	return distgo.ProductParam{
 		ID:                     productID,
+		Name:                   cfg.ProductName,
```

Comment:
I think it will be much cleaner/clearer if `Name` defaults to `ID` if unspecified. 

```suggestion
		Name:                   getConfigStringValue(cfg.Name, defaultCfg.Name, productID),
```